<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeLayout</class>
 <widget class="QDialog" name="homeLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>574</y>
     <width>92</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>@Easy Bensin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonFeedback">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#PushButtonFeedback
{ 
background-color: transparent; 
border-image: url(:/picture/feedback.png); 
background: none; 
border: none; 
background-repeat: none; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonMaps">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#PushButtonMaps{
background-color: qradialgradient(spread:pad, cx:0.499807, cy:0.489, radius:0.5, fx:0.499, fy:0.488909, stop:0.0795455 rgba(0, 147, 185, 255), stop:1 rgba(30, 30, 30, 255));
border: 5px solid black;
border-radius: 40px;
}</string>
   </property>
   <property name="text">
    <string>Maps</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonHistory">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#PushButtonHistory
{ 
background-color: transparent; 
border-image: url(:/picture/history.png); 
background: none; 
border: none; 
background-repeat: none; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonHome">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">
#PushButtonHome
{ 
background-color: transparent; 
border-image: url(:/picture/off.png); 
background: none; 
border: none; 
background-repeat: none; 
}
#PushButtonHome:pressed
{
 border-image: url(:/picture/on.png); 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonAbout">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#PushButtonAbout
{ 
background-color: transparent; 
border-image: url(:/picture/about.png); 
background: none; 
border: none; 
background-repeat: none; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonLevel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#PushButtonLevel
{ 
background-color: transparent; 
border-image: url(:/picture/level.png); 
background: none; 
border: none; 
background-repeat: none; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonReboot">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reboot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
